docs/rest-api/getting-started.md

```diff
@@ -0,0 +1,92 @@
+
+We'll use both [Postman](https://www.getpostman.com/) and [Insomnia](https://insomnia.rest/) clients in these examples. Both are free and will help you visualise what the API offers.
+
+Before proceeding, please read the [REST API docs on authentication which covers the important parts concerning API Keys and Auth](https://woocommerce.github.io/woocommerce-rest-api-docs/#authentication). We're only covering connecting over HTTPS here since it's the simplest and most secure method. You should avoid HTTP if possible.
+
+## Generate Keys
+
+To start using REST API, you first need to generate API keys.
+
+1. Go to *WooCommerce > Settings > Advanced*
+2. Go to the *REST API* tab and click *Add key*.
+3. Give the key a description for your own reference, choose a user with access to orders etc, and give the key *read/write* permissions.
+4. Click *Generate api key*.
+5. Your keys will be shown - do not close this tab yet, the secret will be hidden if you try to view the key again.
+
+![Generated API Keys](_media/keys.png)
+
+## Make a basic request
+
+The request URL we'll test is `wp-json/wc/v3/orders`. On localhost the full URL may look something like this: `https://localhost:8888/wp-json/wc/v3/orders`. Modify this to use your own site URL.
+
+In Postman, you need to set the fields for request type, request URL, and the settings on the authorization tab. For Authorization, choose *basic auth* and enter your *consumer key* and *consumer secret* keys from WooCommerce into the username and password fields
+
+Once done, hit send and you'll see the JSON response from the API if all worked well. You should see something like this:
+
+![Generated API Keys](_media/postman.png)
+
+Insomnia is almost identical to Postman; fill in the same fields and again use basic auth.
+
+![Insomnia](_media/insomnia.png)
+
+Thats it! The API is working.
+
+If you have problems connecting, you may need to disable SSL verification - see the connection issues section below.
+
+## Common connection issues
+
+### Connection issues with localhost and self-signed SSL certificates
+
+If you're having problems connecting to the REST API on your localhost and seeing errors like this:
+
+![SSL Error](_media/sslerror.png)
+
+You need to disable SSL verification. In Postman you can find this in the settings:
+
+![Postman settings](_media/postman-ssl.png)
+
+Insomnia also has this setting the preferences area:
+
+![Insomnia settings](_media/insomnia-ssl.png)
+
+### 401 Unauthorized
+
+Your API keys or signature is wrong. Ensure that:
+
+- The user you generated API keys for actually has access to those resources.
+- The username when authenticating is your consumer key.
+- The password when authenticating is your consumer secret.
+- Make a new set of keys to be sure.
+
+If your server utilizes FastCGI, check that your [authorization headers are properly read](https://support.metalocator.com/en/articles/1654091-wp-json-basic-auth-with-fastcgi).
```

Comment:
This link is broken. If there's no clear alternative here, we could probably link to the Archive.org version of this webpage: https://web.archive.org/web/20230330133128/https://support.metalocator.com/en/articles/1654091-wp-json-basic-auth-with-fastcgi

Reply:
Good catch! Using archive.org seems ok to me. Updated in 4b603c5ed9